File: go/hello-world/hello_world.go
package greeting

// HelloWorld greets the world.
func HelloWorld() string {
	return "Hello, World!"
}

func Gcd(p int, q int) int {
	if q == 0 {
		return p
	}

	r := p % q
	return Gcd(q, r)
}

// fmt.Println()

File: cpp/hello-world/hello_world.cpp
#include "hello_world.h"
#include <iostream>
#include <string>

using namespace std;

namespace hello_world
{
  string hello()
  {
    return "Hello, World!";
  }

  int main() {
    cout << hello_world::hello() << endl;
    return 0;
  }
}
